.github/workflows/add-preview-domain.yml

```diff
@@ -0,0 +1,57 @@
+name: Assign preview-domain
+on:
+  # Trigger when a pull request is opened or reopened
+  pull_request:
+    types: [opened, reopened]
```

Comment:
Just as precaution, can we somehow limit to check if team member opened the PR? Someone could spam a lot of PRs and register many domains in our Vercel account.

Reply:
Our plan has unlimited domains, but if it is easy to somehow check it would be great.

Reply:
We can protect it with this setting
![image](https://github.com/dailydotdev/apps/assets/1681525/d21888ae-06fd-4549-bd59-58cc512e407c)

But I'm pretty sure I can whip something together to ensure the author is from our team/ github org

Reply:
Yes, the above would also work as a first measure 👌

Reply:
Changed the setting :)